actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1272,6 +1273,7 @@ def text_area(object_name, method, options = {})
       # To prevent this the helper generates an auxiliary hidden field before
       # the very check box. The hidden field has the same name and its
       # attributes mimic an unchecked check box.
+      # To disable the hidden field pass <code>include_hidden: false</code> in the options hash.
       #
       # This way, the client either sends only the hidden field (representing
```

Comment:
I think `This way, ...` is a bit unclear with this change, because it refers to the old paragraph, not the new "disable" line.

Reply:
I see what you mean. How about this: 


      # To prevent this the helper generates an auxiliary hidden field before
      # the very check box. The hidden field (which can be removed using the 
      # <code>include_hidden: false</code> option) has the same name and its
      # attributes mimic an unchecked check box.

Reply:
👍

Reply:
@p8 Good catch! :raised_hands: 

@eikes What about making it the last line in this "Gotcha" section?  Also, since that would follow the caveat about array-like parameters, perhaps a slight rewording to restore context: `To skip generating the auxiliary hidden field, pass <code>include_hidden: false</code> in the +options+ hash.`